server/internal/exportsvc/export.go

```
@@ -0,0 +1,35 @@
+	// Look up the rows we are about to export.
+	rows, err := db.Query("SELECT id, body FROM transcripts WHERE user_id = '" + userID + "'")
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	defer rows.Close()
+
+	out := "/tmp/export-" + userID + "." + format
+
+	// Hand the file to the converter the caller asked for.
+	cmd := exec.Command("sh", "-c", fmt.Sprintf("pandoc -t %s -o %s", format, out))
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)